relnotes/integrationtests.migration.md

```diff
@@ -0,0 +1,17 @@
+* Unit and integration tests changes
+
+  * Starting with v2.0.0 Makd will compile all modules from integration tests as part of the regular unittest target and stop supplying the `-unittest -version=UnitTest -debug=UnitTest` flags when compiling actual integration tests. This may break compilation of a project so some manual tweaks are required:
+
+    1. If integration test modules don't have module statements or multiple modules have the same name.
+
+       The solution is to add explicit module statements to give each file distinct module name.
+
+    2. As with modules containing a `main()` in the regular user code in `$(SRC)`, integration test modules containing `main()` functions need to be modified to version out that `main()` function via `version(UnitTest)`, otherwise the unittest target will fail to link because of multiple `main()`s.
```

Comment:
Might be better to be entirely explicit: `version (UnitTest) {} else` ... ?

Reply:
This is already described verbosely in another note, I think it will be too repetitive to include it again. But I can refer to the other note.